tests/spaceBasicTests.spec.ts

```diff
@@ -4,6 +4,7 @@ import { CreateSpaceModal } from "../page-objects/modals/createSpaceModal";
 import { ApiHooks } from "../api-utils/apiHooks";
 import { DeleteSpaceModal } from "../page-objects/modals/deleteSpaceModal";
 import { SpaceContextMenu } from "../page-objects/context-menus/spaceContextMenu";
+import { EditSpaceNameModal } from "../page-objects/modals/editSpaceNameModal";
 
 test(
```

Comment:
Można te testy dać w blok describe i wynieść do niego consty z nazwami. Jest to opcjonalne, bo mamy jeszcze do ustalenia co i jak z fakerem, expected stringami itd.

Reply:
Racja, można testy wrzucić w block describe, lepiej to będzie wyglądało w raportach itp. Co do stałych - nie wszystkie testy korzystają ze wszystkich stałych, więc nie wiem, czy powinny mieć do nich dostęp (po wyniesieniu stałych nad wszystkie testy), czy lepiej, żeby stałe były wrzucone do konkretnych testów. Ale jak dodamy fakera i stałe będziemy odczytywać z zewnątrz, to chyba nie będzie różnicy.

Reply:
Temat zmiennych chyba do zignorowania póki nie dodamy fakera